packages/sql-openapi/test/simple.test.js

```diff
@@ -139,13 +148,15 @@ test('simple db, simple rest API', async (t) => {
   {
     const res = await app.inject({
       method: 'POST',
-      url: '/pages/1?fields=title',
+      url: '/pages/1?fields=title,author',
```

Comment:
I cannot find the test for the bug, e.g.: `/pages/1?fields=title` (with "missing" author), that should pass and return just the titles.

Reply:
Hi, tests were already there, they're not failing any more. 
https://github.com/platformatic/platformatic/blob/94da3151e3f10c89884cf0099dc5a32cabcf0ee6/packages/sql-openapi/test/simple.test.js#L129-L137

As per what I understand, this change should only affect the `GET` requests, and that's where I've changed the schemas, all GET requests now, have no required fields in their schema. since tests were already there for the `title`, after adding `author` they shouldn't fail, and that's the case now.

Reply:
The thing that I don't understand is,

Reply:
I don't understand why `DELETE` and `POST` schemas can have fields params. and why all of them are required?
most of the changes in tests here, are to update these fields in those requests as the one you've highlighted here. 

is it intended? what is the purpose then? or it's a bug?

Reply:
OK, now I see, you added `author`, so  [this test](https://github.com/platformatic/platformatic/blob/94da3151e3f10c89884cf0099dc5a32cabcf0ee6/packages/sql-openapi/test/simple.test.js#L129-L137) started failing and then your change fixed. 
About the `DELETE` and `POST`, they are returning the affected record, and `fields` should specify what fields you want in return. I think we should do the same that we did for GET to avoid the same problem OR -maybe better- remove the `fields` feature for `DELETE` and `POST`. Maybe @mcollina has an opinion on this.

Reply:
I think, for `POST`, and `DELETE` requests ( `pages/1` ), fields shouldn't even be allowed, no need for it. 
and even more, for the `POST` request (`pages/1` ), there should be no required fields,  to allow partial update, what I'm experiencing now is if I want to update the title, I need to also send the author value

Reply:
That's great, yes. I've just written my comment before seeing yours. so we're on the same side I think